docs/hosting.md

```diff
@@ -161,3 +161,123 @@ kubectl get service
 *** Will be covered later
 
 
+## Amazon EC2
+
+- Visit [AWS Portal](https://aws.amazon.com/free/).
+
+- Log in.
+If you dont have an account set it up. It will ask you for **Credit Card details**.  <br><br>
```

Comment:
Do not use line break <br>. At times markdown conversion breaks due to this

Reply:
Okk sir